### ISSUE TITLE
Fluent Store 0.2.1 crashes silently if Visual C++ Redistributable is not installed

### ISSUE THREAD

**Issue by LightWayUp** [yoshiask/FluentStore#27]
I unintentionally discovered this when testing on a clean instance (Windows Sandbox)

Fluent Store 0.2.1 crashes silently upon launch, if Visual C++ Redistributable isn't installed.  
In the Event Viewer, 2 error events are shown:

Level | Source | Event ID | Task Category
--- | --- | --- | ---
Error | Application Error | 1000 | (100)
Error | .NET Runtime | 1026 | None

Application Error:
> Faulting application name: FluentStore.App.exe, version: 0.2.1.0, time stamp: 0x61735063  
> Faulting module name: KERNELBASE.dll, version: 10.0.21390.1000, time stamp: 0xc6d36e55  
> Exception code: 0xe0434352  
> [...]  
> Faulting application path: C:\Program Files\WindowsApps\52374YoshiAskharoun.FluentStore_0.2.1.0_x64__bcem08bwhrc72\FluentStore.App.exe  
> Faulting module path: C:\Windows\System32\KERNELBASE.dll  
> [...]  
> Faulting package full name: 52374YoshiAskharoun.FluentStore_0.2.1.0_x64__bcem08bwhrc72  
> Faulting package-relative application ID: App

.NET Runtime:
> Application: FluentStore.App.exe  
> CoreCLR Version: 6.0.21.52210  
> .NET Version: 6.0.0  
> Description: The process was terminated due to an unhandled exception.  
> Exception Info:
> ```
> System.DllNotFoundException: Unable to load DLL 'Microsoft.ui.xaml.dll' or one of its dependencies: The specified module could not be found. (0x8007007E)
>    at FluentStore.Program.XamlCheckProcessRequirements()
>    at FluentStore.Program.Main(String[] args)
> ```

Searching online leads to [microsoft/WindowsAppSDK: .NET 6 and WINUI3 fails to start Unable to load DLL 'Microsoft.ui.xaml.dll' #1762](https://github.com/microsoft/WindowsAppSDK/issues/1762), with [this comment](https://github.com/microsoft/WindowsAppSDK/issues/1762#issuecomment-966649362) giving the suggestion to install VCRedist manually, which in this case does indeed work.

This issue didn't happen with the previous release 0.1.2, because unlike that version, 0.2.1 doesn't declare `Microsoft.VCLibs` as a dependency, and the only dependency `Microsoft.WindowsAppRuntime.1.0` doesn't declare any dependencies either. Thus, Windows Update wouldn't automatically download the required Visual C++ Redistributable.

0.2.1 AppxManifest.xml:
```xml
<Dependencies>
  <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
  <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17763.0" MaxVersionTested="10.0.22000.0" />
  <PackageDependency Name="Microsoft.WindowsAppRuntime.1.0" MinVersion="0.319.455.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
</Dependencies>
```

0.1.2 AppxManifest.xml:
```xml
<Dependencies>
  <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0"/>
  <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0"/>
  <PackageDependency Name="Microsoft.UI.Xaml.2.6" MinVersion="2.62107.6002.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"/>
  <PackageDependency Name="Microsoft.NET.Native.Framework.2.2" MinVersion="2.2.29512.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"/>
  <PackageDependency Name="Microsoft.NET.Native.Runtime.2.2" MinVersion="2.2.28604.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"/>
  <PackageDependency Name="Microsoft.VCLibs.140.00" MinVersion="14.0.30035.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"/>
</Dependencies>
```

**Comment by yoshiask** [comment]
This issue may be fixed in v0.3.0-beta, would you be able to test this again?

**Comment by LightWayUp** [comment]
Thanks, I tested v0.3.0-beta release and it works! 🎊
